web/src/app/interfaces/CharState.ts

```diff
@@ -0,0 +1,5 @@
+export default interface CharState {
```

Comment:
When the game text is rendered, each character will have a state assigned to it which is how we determine what colour of text to render.

Currently index is not being used, but may be used in the future.